### DISCUSSION TITLE
Why is my GPS data always close to 0？

### DISCUSSION THREAD

**Discussion by ly1109114191** [carla-simulator/carla#4806] [Q&A]
Why is my GPS data always close to 0？For example, the longitude, latitude and altitude obtained from GNSS sensors are all like this，（0.00142560357436138，-0.000986275847799022，2.94161367416381）.
I've reloaded Carla twice, and that's the result.

**Comment by matejm42** [comment]
Because CARLA GNNS compustes coodrinates from distance from point (0,0) on map in CARLA. That point is (0,0) also in GNSS. And because of the maps in CARLA are relatively small and 1° is approximately 111km, you can't get from (0,0) far enough to have even 0,1° lat/lon.

More here https://stackoverflow.com/questions/4102520/how-to-transform-a-distance-from-degrees-to-metres[](https://stackoverflow.com/questions/4102520/how-to-transform-a-distance-from-degrees-to-metres)
